lib/katello/tasks/jenkins.rake

```diff
@@ -2,8 +2,6 @@ require File.expand_path("../engine", File.dirname(__FILE__))
 
 begin
   namespace :jenkins do
-    ENV['USE_MEAN_TIME_REPORTER'] = '1' unless ENV['USE_MEAN_TIME_REPORTER'] == '0'
```

Comment:
@jturel do you remember why this env var was introduced, instead of running the reporter all the time?

In Foreman we now implemented it without this var, but I wanted to check back if there was any reason to want this disabled?

Reply:
Oh, I'm not actually sure. Though I don't think it was ever set to 0 until later on when we found the mean time reporter somehow causing timeouts in jenkins - or at least that was the theory at the time.

Reply:
Yeah, I don't think we ever made Jenkins or anything else set it to 0.